pages/pipelines/notifications.md.erb

```diff
@@ -0,0 +1,99 @@
+# Triggering Notifications
+
+The `notify` attribute allows you to trigger build notifications to different services. You can also choose to conditionally send notifications based on pipeline events like build status.
+
+<%= toc %>
+
+Add notifications to your pipeline with the `notify` attribute. This sits  at the same level as `steps` in your `pipeline.yml` file.
+
+The below example will send a notification email every time a build is created on this pipeline: 
+
+```yaml
+steps:
+  - command: "tests.sh"
+  - wait
+  - command: "build.sh"
+
+notify:
+  - email: "dev@acmeinc.com"
+```
+
+You can trigger notifications as emails or as a message in a Basecamp Campfire.
+
+## Conditional notifications
+
+You can choose to only trigger notifications when a statement evaluates to true using the `if` attribute.
+
+In the below example the email notification will only be triggered if the build passes:
+
+```yaml
+notify:
+  - email: "dev@acmeinc.com"
+    if: "build.status == passed"
+```
+
+## Email
+
+Add an email notification to your pipeline using the `email` attribute of the `notify` yaml block:
+
+```yaml
+notify:
+  - email: "dev@acmeinc.com"
+```
+
+The `email` attribute accepts a single email address as a string. To send notifications to more than one address, add each address as a separate email notification attribute:
+
+```yaml
+notify:
+  - email: "dev@acmeinc.com"
+  - email: "sre@acmeinc.com"
+  - email: "qa@acmeinc.com"
+```
+
+## Basecamp Campfire Message
+
+To send notifications to a Basecamp Campfire, you'll need to set up a chatbot in Basecamp as well as adding the notification to your `pipeline.yml` file. Basecamp admin permission will be required to setup your chatbot.
+
+1. Add a [chatbot](https://m.signalvnoise.com/new-in-basecamp-3-chatbots/) to the Basecamp project or team that you'll be sending notifications to. 
+1. Set up your chatbot with a name and an optional URL. If you'd like to include an image, you can find the Buildkite logo in our [Brand Assets](https://buildkite.com/brand-assets).
+1. On the next page of the chatbot setup, copy the URL that Basecamp provides in the `curl` code snippet
+1. Add a Basecamp notification to your pipeline using the `basecamp_campfire` attribute of the `notify` yaml block and the URL copied from your Basecamp chatbot:
+
+```yaml
+notify:
+  - basecamp_campfire: URL
+```
+
+The `basecamp_campfire` attribute accepts a single URL as a string.
+
+## Slack Channel Message
+
+Add a Slack notification to your pipeline using the `slack` attribute of the `notify` yaml block:
+
+```yaml
+notify:
+  - slack: #general
+```
+
+The above example will deliver the notification to all configured workspaces that have a channel named 'general'.
```

Comment:
I wonder if we should be more explicit that workspaces have to be configured in the organisation notification services?:

https://buildkite.com/organizations/ellie-test-organization/services